docs/src/guide/installation.md

```diff
@@ -49,6 +48,12 @@ well:
 	brew install arm-gcc-bin
 	brew install openocd --HEAD
 
+scons now works with Python 3. Unfortunately, macOS still defaults to Python 2.
+The less intrusive way to run all scons scripts with Python 3 add this to your
+~/.bashrc or ~/.zshrc
+
+	alias scons="/usr/bin/env python3 $(which scons)"
```

Comment:
This is giving me an issue with `tools/scripts/examples_compile.py`:
```
Building: game_of_life with SCons
scons: Reading SConscript files ...
scons: done reading SConscript files.
scons: Building targets ...
scons: building terminated because of errors.scons: *** [modm/src/modm/architecture/detect.hpp] ValueError : unsupported pickle protocol: 4
```
Can you confirm? Maybe this is just my local issue.

Reply:
Running `scons` via zsh alias works fine.

Reply:
I cannot confirm this error. I can compile the game of life from the directory manually.

But the script does not inherit the alias, so it seems that scons again uses Python2.

Reply:
It might need
```python
commands.append("python3 `which scons` build --cache-show --random")
```

Reply:
Yep, that fixes it, great!